db-commons/src/main/resources/reference.conf

```diff
@@ -2,6 +2,36 @@ bitcoin-s {
     datadir = ${HOME}/.bitcoin-s
     network = regtest # regtest, testnet3, mainnet
 
+    logging {
```

Comment:
is there any way we can log everything to one file if we want to?

Reply:
WYM? Right now all loggers log to the same file

Reply:
oh -- i thought they were being logged to different files in this PR. In the future they will be right? In that case, can we somehow having a setting to dump everything into one file?

Reply:
It's trivial to add new log files for specific loggers, as well as get one logger to output to several files if we want that in the future.